source/content.js

```diff
@@ -168,6 +169,10 @@ async function onDomReady() {
 		enableFeature(autoLoadMoreNews);
 	}
 
+	if (pageDetect.isUserProfile()) {
+		enableFeature(setDefaultRepositoriesTypeToSources);
+	}
+
```

Comment:
If it’s inside ajaxedPagesHandler it’s not needed here

Reply:
Done